mentat/code_context.py

```diff
@@ -148,12 +148,13 @@ async def get_code_message(
             "\n".join(include_files_message), model, full_message=False
         )
 
-        tokens_used = (
-            prompt_tokens + meta_tokens + include_files_tokens + config.token_buffer
-        )
+        tokens_used = prompt_tokens + meta_tokens + include_files_tokens
         if not is_context_sufficient(tokens_used):
-            raise ContextSizeInsufficient()
```

Comment:
Should we print something here so the user knows what the problem is?

Reply:
We do print something in is_context_sufficient; maybe we should move the warning out of the function, but we do use it in multiple places.